### PR TITLE
Fix Crash: Cascade with Jetpack Compose 1.5.0+

### PR DESCRIPTION
**Pull request by kasem-sm**
Latest alpha release of Material3 transitively updates Compose to 1.5.0-alpha04 and it has some breaking changes, hence the crash (mentioned in #36).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,10 +4,10 @@ buildscript {
   ext.versions = [
     minSdk: 23,
     compileSdk: 33,
-    kotlin: '1.7.20',
-    composeUi: '1.3.1',
-    composeCompiler: '1.4.0-alpha01',
-    composeUiMaterial3: '1.0.1',
+    kotlin: '1.8.21',
+    composeUi: '1.5.0-alpha04',
```
Comment (saket) [inline build.gradle]: Let's depend on `1.5.0-alpha01`. Not all projects may be ready to consume newer alpha versions yet.

File: build.gradle [hunk 2]
```diff
@@ -4,10 +4,10 @@ buildscript {
...
+    composeCompiler: '1.4.7',
+    composeUiMaterial3: '1.2.0-alpha01',
```
Comment (saket) [inline build.gradle]: Any reason for depending on an alpha version of material3? Can we depend on `1.1.0` instead?

File: build.gradle [hunk 3]
```diff
@@ -35,6 +35,6 @@ allprojects {
   }
 }
 
-task clean(type: Delete) {
+tasks.register('clean', Delete) {
```
Comment (saket) [inline build.gradle]: This is unrelated to this PR. Can you remove this?

**Review by saket — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (kasem-sm) [reply build.gradle]: Agreed.

File: build.gradle (hunk 3 above)
Comment (kasem-sm) [reply build.gradle]: Sure.

File: build.gradle (hunk 2 above)
Comment (kasem-sm) [reply build.gradle]: Yep. 1.1.0 works as well.

**Review by saket — APPROVED** [review]

**Comment by saket** [comment]
Thank you!

**Comment by kasem-sm** [comment]
@saket, we need to update Material3 to 1.2.0-alpha02.

**Comment by saket** [comment]
@kasem-sm Thinking of updating compose to 1.5.0-beta01 instead. What do you think?

**Comment by kasem-sm** [comment]
@saket, a bit unsure atm. Gotta try in a sample project and let you know asap.
